### DISCUSSION TITLE
Repeat MACE on QM9

### DISCUSSION THREAD

**Discussion by yesheng22** [ACEsuit/mace#366] [Q&A] [answered]
Since the performance of MACE is great according to "Evaluation of the MACE force field architecture: From medicinal chemistry to materials science". But the input file is not provided in (https://macedocs.readthedocs.io/en/latest/examples/training_examples.html). 
What is the most suitable format for QM9 data files to be connected to the code? I am currently using the format of xyz files with force all 0, which seems to mislead the model.

**Comment by davkovacs** [comment, marked as answer]
Make sure to use the correct units for energy related quantities. eV for energies and A for positions / forces. And also make sure to use correct E0-s, by setting average.
